code/modules/antagonists/xeno/xeno.dm

```diff
@@ -58,3 +58,12 @@
 	..()
 	if(!mind.has_antag_datum(/datum/antagonist/xeno))
 		mind.add_antag_datum(/datum/antagonist/xeno)
+
+/mob/living/carbon/alien/on_wabbajacked(mob/living/new_mob)
+	. = ..()
+	if(!mind)
+		return
+	if(isalien(new_mob))
+		return
+	mind.remove_antag_datum(/datum/antagonist/xeno)
+	mind.special_role = null
```

Comment:
Should we do this before calling the parent, just in case the the new mob does stuff to the mind, now or in the future?

Reply:
it is only for the purpose of removing the bad antag assignment you get from rapidly changing forms through wabbajack